rotkehlchen/db/accounting_rules.py

```diff
@@ -32,7 +44,7 @@ def add_accounting_rule(
             event_subtype: HistoryEventSubType,
             counterparty: Optional[str],
             rule: 'BaseEventSettings',
-    ) -> None:
+    ) -> int:
```

Comment:
add to the docstring that the identifier of the rule is returned by the function